File: Project1-code/chapter3-work/code-Python/while-loop.py
def pow(a, b):
    if(b==0):
        return 1
    else:
        res = 1
        while(b!=0):
            res *= a
            b = b - 1
        return res

n = int(input())
x = float(input())
i = 1
sum = 0

while(i<=n):
    sum += pow(-1, i-1)*pow(x, i)
    i = i + 1

print(sum)

File: Project1-code/chapter3-work/code-C/while-loop.cpp
#include <iostream>
using namespace std;
double pow(double a, int b) {
    if(b==0) return 1;
    double res = 1;
    while(b!=0) {
        res *= a;
        b--;
    }
    return res;
}
int main(){
    int n, i=1;
    double x, sum = 0;
    cin >> n >> x;
    while (i<=n) {
        sum += pow(-1, i-1)*pow(x, i);
        i++;
    }
    cout << sum;
    return 0;
}